elastic/kibana
# [Securitysolution] Add Risk score missing privileges callout to enablement flyout

## Summary

The entity analytics enablement model doesn't show the missing privileges warning for the Entity Risk score.

To fix it, I added to the model the same callout we display on the risk engine page.

### What is not included
* Improvement to the risk engine callout
* Fix risk engine callout bugs https://github.com/elastic/security-team/issues/11138

### How to test it
* Create a user with no privileges except to the security solution app and the `logs*` index
* Login and open the entity analytics page with the non-privileged user
* Click enable and check if the model displays the missing privileges callout for the risk engine

### Checklist
